tests/services/phase.service.spec.ts

```diff
@@ -0,0 +1,119 @@
+import { of } from 'rxjs';
+import { SessionData } from '../../src/app/core/models/session.model';
+import { UserRole } from '../../src/app/core/models/user.model';
+import { Phase, PhaseService } from '../../src/app/core/services/phase.service';
+import { UserService } from '../../src/app/core/services/user.service';
+
+const testStudent = {
+  loginId: 'testStudent',
+  role: UserRole.Student
+};
+
+const testTutor = {
+  loginId: 'testTutor',
+  role: UserRole.Tutor
+};
+
+const mockSettingsFile: {} = {
+  'openPhases' : [Phase.phaseBugReporting],
+  'phaseBugReporting': 'bugreporting',
+  'phaseTeamResponse': 'pe-results',
+  'phaseTesterResponse': 'testerresponse',
+  'phaseModeration': 'pe-evaluation'
+};
+
+const mockSessionData: SessionData = mockSettingsFile as SessionData;
+
+let phaseService: any;
+let githubService: any;
+let userService: any;
+
+describe('PhaseService', () => {
+    beforeEach(() => {
+      githubService = jasmine.createSpyObj('GithubService',
+        ['fetchSettingsFile', 'storePhaseDetails', 'createRepository']);
+      userService = new UserService(null, null);
+      phaseService = new PhaseService(null, githubService, null, userService, null);
+    });
+
+    describe('.fetchSessionData()', () => {
+      it('should fetch the correct settings file', () => {
+        githubService.fetchSettingsFile.and.returnValue(of(mockSettingsFile));
+        phaseService.fetchSessionData().subscribe((sessionData: SessionData) => {
+          expect(sessionData).toEqual(mockSessionData);
+        });
+      });
+    });
+
+    describe('.updateSessionParameters()', () => {
+      it('should update the currentPhase and repoName based on given sessionData', () => {
+        githubService.storePhaseDetails.and.callFake(() => {});
+        phaseService.updateSessionParameters(mockSessionData);
+
```

Comment:
Should we remove the line here? Cause its not in the other functions also 😅

Reply:
Yes yes! I believe you are referring to the empty line, am i right? :O

Reply:
Ahaha yeah the empty line 😅

Reply:
Alright!